src/inputComponent.js

```diff
@@ -16,10 +42,21 @@ class InputComponent extends React.Component {
   };
 
   render() {
-    return <React.Fragment>
-      <input onChange={this.handleInputChange} id={"submit"} value={this.state.value}/>
-      <button onClick={()=>this.onSubmit()}>Submit</button>
-    </React.Fragment>
+    return <Paper style={styles.root} elevation={1}>
+      <InputBase
+        style={styles.input}
+        autoComplete='off'
+        placeholder={"Введите заметку"}
+        onChange={this.handleInputChange}
+        id={"submit"}
+        value={this.state.value}
+      />
+      <Divider style={styles.divider}/>
+      <IconButton style={styles.button} onClick={() => this.onSubmit()}>
```

Comment:
each props from new line
```suggestion
      <IconButton 
          style={styles.button} 
          onClick={() => this.onSubmit()}
      >
```